new-log-viewer/package.json

```diff
@@ -81,6 +83,24 @@
           "yscope/typescript"
         ]
       }
-    ]
+    ],
+    "rules": {
+      "no-restricted-imports": [
```

Comment:
This is causing errors for import `SvgIconComponents` and `DefaultColorPalette`. Should we revert this change, since it's blocking importing types directly?